src/components/simulation/SimulationPanel.svelte

```diff
@@ -95,13 +97,62 @@
     effects.updateSimulation(newSimulation, newFiles);
   }
 
-  async function onChangeSimulationTemplate(event: Event) {
-    const { value } = getTarget(event);
-    const id = value as number;
-    const template = { ...$simulation?.template, id };
-    const newSimulation: Simulation = { ...$simulation, template };
+  function onResetFormParameters(event: CustomEvent<FormParameter>) {
```

Comment:
For the future, how would we show simulation arguments from historical simulations?

Reply:
@camargo do you mean how we should reset the value to the value in the historical simulation?

Reply:
Yeah